pkgs/development/libraries/bobcat/default.nix

```diff
@@ -24,6 +24,9 @@ stdenv.mkDerivation rec {
   postPatch = ''
     substituteInPlace INSTALL.im --replace /usr $out
     patchShebangs .
+  '' + lib.optionalString (with stdenv.targetPlatform; isAarch64 && isLinux) ''
+    # have to link to static gcc lib on aarch64-linux explicitly
+    substituteInPlace icmake/libraries --replace 'g_sharedLibReq' 'g_sharedLibReq + " -lgcc"'
```

Comment:
Are you sure this isn't better done through `NIX_CFLAGS_LINK` or `NIX_LDFLAGS`?

Reply:
I had another problem where the `$NIX_*` variables didn't work, so I was probably reluctant to apply them here... Fixed now.